### PR TITLE
reports: improve inventory list for items

### PR DESCRIPTION
**Pull request by lauren-d**
reports: improve inventory list for items

* Changes API endpoint for inventory list.
* Adapts configuration for export formats into item route.

Co-Authored-by: Lauren-D <laurent.dubois@itld-solutions.be>

## Dependencies

My PR depends on `rero-ils#<xx>`'s PR(s):

* rero/rero-ils#1842
* rero/ng-core#384

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: projects/admin/src/app/routes/items-route.ts [hunk 1]
```diff
@@ -134,8 +134,10 @@ export class ItemsRoute extends BaseRoute implements RouteInterface {
             exportFormats: [
               {
                 label: 'CSV',
-                format: 'csv'
-              }
+                format: 'csv',
+                endpoint: this._routeToolService.apiService.endpointPrefix + '/item/inventory',
```
Comment (Garfield-fr) [inline projects/admin/src/app/routes/items-route.ts:138]: `this._routeToolService.apiService.getEndpointByType('item/inventory')` ?

**Review by Garfield-fr — APPROVED** [review]

**Review by zannkukai — APPROVED** [review]

File: projects/admin/src/app/routes/items-route.ts (hunk 1 above)
Comment (lauren-d) [reply projects/admin/src/app/routes/items-route.ts]: ok, done

File: projects/admin/src/app/routes/issues-route.ts [hunk 2]
```diff
@@ -71,7 +71,9 @@ export class IssuesRoute extends BaseRoute implements RouteInterface {
             exportFormats: [
               {
                 label: 'CSV',
-                format: 'csv'
+                format: 'csv',
+                endpoint: this._routeToolService.apiService.getEndpointByType('item/inventory'),
+                disableMaxRestResultsSize: true,
```
Comment (sebastiendeleze) [inline projects/admin/src/app/routes/issues-route.ts:76]: This property seems not to be used

**Review by sebastiendeleze — APPROVED** [review]

File: projects/admin/src/app/routes/issues-route.ts (hunk 2 above)
Comment (lauren-d) [reply projects/admin/src/app/routes/issues-route.ts:76]: this is used in PR on ng-core (https://github.com/rero/ng-core/pull/384).

**Review by iGormilhit — APPROVED** [review]

**Pull request merged**